gym/f110_gym/envs/base_classes.py

```diff
@@ -26,6 +26,7 @@
 Replacement of the old RaceCar, Simulator classes in C++
 Author: Hongrui Zheng
 """
+from typing import List
```

Comment:
Starting with Python 3.7, this allows to use the standard collections directly for type hinting.

```suggestion
from __future__ import annotations
```